modules/util/portal-tools-bundle-support/src/main/java/com/liferay/portal/tools/bundle/support/internal/util/HttpUtil.java

```diff
@@ -177,7 +177,7 @@ private static Path _downloadFile(
 
 				if (index > 0) {
 					fileName = dispositionValue.substring(
-						index + 10, dispositionValue.length() - 1);
+						index + "filename=".length());
```

Comment:
hey @Seiphon  could you sent a unit test for this problem?  I would like to see a URL that breaks the original code and show how this fix is needed.  Let me know if you need to help setting up a unit test for this (setting up a mock/test http server).

Reply:
OK, will send a unit test for this issue.